reverse-proxy.md

```diff
@@ -145,15 +145,15 @@ https://<your-nc-domain>:443 {
     reverse_proxy localhost:11000
 }
 ```
-The Caddyfile is a text file called `Caddyfile` (no extension) which – if you should be running Caddy inside a container – should usually be created in the same location as your `compose.yaml` file prior to starting the container.
+The Caddyfile is a text file called `Caddyfile` (no extension) which – if you should be running Caddy inside a container – should usually be created in the same location as your `docker-compose.yaml` file prior to starting the container.
```

Comment:
Since compose files are used by [multiple backends](https://github.com/compose-spec/compose-spec#implementations), `compose.yaml` is more relevant. See:
https://github.com/compose-spec/compose-spec/blob/231b09c30d339e950c0da17fe5bdc793366b8fde/03-compose-file.md?plain=1#L11-L13

Reply:
```suggestion
The Caddyfile is a text file called `Caddyfile` (no extension) which – if you should be running Caddy inside a container – should usually be created in the same location as your `compose.yaml` file prior to starting the container.
```